internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -484,67 +517,68 @@ func (r *ClusterDeploymentsReconciler) syncManifests(ctx context.Context, cluste
+func (r *ClusterDeploymentsReconciler) getClusterDeploymentManifest(ctx context.Context, clusterInstall *hiveext.AgentClusterInstall) (map[string]string, error) {
 	configuredManifests := &corev1.ConfigMap{}
 	configuredManifests.Data = map[string]string{}
 	// get manifests from configmap if we have reference for it
-	if clusterDeployment.Spec.Provisioning.ManifestsConfigMapRef != nil {
-		err := r.Get(ctx, types.NamespacedName{Namespace: clusterDeployment.Namespace,
-			Name: clusterDeployment.Spec.Provisioning.ManifestsConfigMapRef.Name}, configuredManifests)
+	if clusterInstall.Spec.ManifestsConfigMapRef != nil {
+		err := r.Get(ctx, types.NamespacedName{Namespace: clusterInstall.Namespace,
+			Name: clusterInstall.Spec.ManifestsConfigMapRef.Name}, configuredManifests)
 		if err != nil {
 			r.Log.WithError(err).Errorf("Failed to get configmap %s in %s",
-				clusterDeployment.Spec.Provisioning.ManifestsConfigMapRef.Name, clusterDeployment.Namespace)
+				clusterInstall.Spec.ManifestsConfigMapRef.Name, clusterInstall.Namespace)
 			return nil, err
 		}
 	}
 	return configuredManifests.Data, nil
 }
 
-func (r *ClusterDeploymentsReconciler) addCustomManifests(ctx context.Context, clusterDeployment *hivev1.ClusterDeployment,
+func (r *ClusterDeploymentsReconciler) addCustomManifests(ctx context.Context, clusterInstall *hiveext.AgentClusterInstall,
 	cluster *common.Cluster) error {
 
 	alreadyCreatedManifests, err := r.Manifests.ListClusterManifestsInternal(ctx, operations.ListClusterManifestsParams{
 		ClusterID: *cluster.ID,
 	})
 	if err != nil {
-		r.Log.WithError(err).Errorf("Failed to list manifests for %q cluster deployment", clusterDeployment.Name)
+		r.Log.WithError(err).Errorf("Failed to list manifests for %q cluster install", clusterInstall.Name)
 		return err
 	}
 
 	// if reference to manifests was deleted from cluster deployment
 	// but we already added some in previous reconcile loop, we want to clean them.
 	// if no reference were provided we will delete all manifests that were in the list
-	if clusterDeployment.Spec.Provisioning.ManifestsConfigMapRef == nil && len(alreadyCreatedManifests) == 0 {
+	if clusterInstall.Spec.ManifestsConfigMapRef == nil && len(alreadyCreatedManifests) == 0 {
 		r.Log.Debugf("Nothing to do, skipping manifest creation")
 		return nil
 	}
 
-	return r.syncManifests(ctx, cluster, clusterDeployment, alreadyCreatedManifests)
+	return r.syncManifests(ctx, cluster, clusterInstall, alreadyCreatedManifests)
 }
 
-func (r *ClusterDeploymentsReconciler) isSNO(cluster *hivev1.ClusterDeployment) bool {
-	return cluster.Spec.Provisioning.InstallStrategy.Agent.ProvisionRequirements.ControlPlaneAgents == 1 &&
-		cluster.Spec.Provisioning.InstallStrategy.Agent.ProvisionRequirements.WorkerAgents == 0
+func (r *ClusterDeploymentsReconciler) isSNO(cluster *hiveext.AgentClusterInstall) bool {
```

Comment:
for consistency with other functions in `/clusterdeployments_controller.go`
`cluster` here should be named `clusterInstall`

Reply:
Done